include/geos/geom/CoordinateSequence.h

```diff
@@ -74,11 +74,11 @@ class GEOS_DLL CoordinateSequence {
     CoordinateSequence();
 
     /**
-     * Create a CoordinateSequence capable of storing XY or XYZ coordinates.
+     * Create a CoordinateSequence capable of storing XY, XYZ or XYZM coordinates.
```

Comment:
For example here, should this also mention XYM ?

Reply:
We can't create an XYM here because we have to interpret `dim = 3` as XYZ for backwards compatibility. Ultimately we will want to deprecate this in favor of the explicit `hasz`, `hasm` constructors.